Tests/Marketplace/upload_packs.py

```diff
@@ -463,6 +463,99 @@ def get_recent_commits_data(content_repo: Any):
     return content_repo.head.commit.hexsha, content_repo.commit('origin/master~1').hexsha
 
 
+def update_index_with_priced_packs(private_storage_bucket: Any, extract_destination_path: str,
+                                   index_folder_path: str, pack_names: set, is_private_build: bool) \
+        -> Tuple[Union[list, list], str, Any]:
+    """ Updates index with priced packs and returns list of priced packs data.
+
+    Args:
+        private_storage_bucket (google.cloud.storage.bucket.Bucket): google storage private bucket.
+        extract_destination_path (str): full path to extract directory.
+        index_folder_path (str): downloaded index folder directory path.
+        pack_names (set): Collection of pack names.
+        is_private_build (bool): Indicates if the build is private.
+
+    Returns:
+        list: priced packs from private bucket.
+
+    """
+    private_index_path = ""
+    private_packs = []
+
+    try:
+        (private_index_path, private_index_blob, _) = \
+            download_and_extract_index(private_storage_bucket,
```

Comment:
This is not compatible with generic bucket, `download_and_extract_index` will use `GCPConfig.STORAGE_BASE_PATH` to construct the `index_storage_path`, that is true only if the bucket is the production bucket but for example for the build bucket this won't work. Can we support the generic case here?

Reply:
@guyfreund We will open a separate PR for this change. The purpose of this PR is to revert a change so we can fix the index in the prod bucket.